x-pack/plugins/upgrade_assistant/public/application/components/overview/fix_logs_step/fix_logs_step.tsx

```diff
@@ -47,13 +48,34 @@ const i18nTexts = {
         'Go to your logs directory to view the deprecation logs or enable log collecting to see them in the UI.',
     }
   ),
+  deniedPrivilegeTitle: i18n.translate(
+    'xpack.upgradeAssistant.overview.deprecationLogs.deniedPrivilegeTitle',
+    {
+      defaultMessage: `You're missing index privileges`,
+    }
+  ),
+  deniedPrivilegeDescription: (privilegesMissing: MissingPrivileges) =>
+    i18n.translate('xpack.upgradeAssistant.overview.deprecationLogs.deniedPrivilegeDescription', {
+      defaultMessage:
+        'To be able to analyze deprecataion logs, you must have {privilegesCount, plural, one {this index privilege} other {these index privileges}}: {missingPrivileges}',
+      values: {
+        missingPrivileges: privilegesMissing?.index?.join(', '),
```

Comment:
Wasn't sure if it would be a good idea to highlight this somehow with either a EuiCode or making it bold:

<img width="1098" alt="Screenshot 2021-09-17 at 21 00 49" src="https://user-images.githubusercontent.com/1191206/133840431-0927176a-3b3d-4385-9e5a-b881eeb20ecc.png">
<img width="1096" alt="Screenshot 2021-09-17 at 21 01 03" src="https://user-images.githubusercontent.com/1191206/133840434-cb4c5878-fd3b-46e9-84e2-4cd339fb6b9d.png">

Reply:
EuiCode's `transparentBackground` prop works well here.

![image](https://user-images.githubusercontent.com/3674406/134046294-172c08c2-0c5f-4c24-b20c-0aa205244b85.png)